src/components/custom/ParameterRowError.js

```diff
@@ -32,15 +32,17 @@ export class ParameterRowError extends Component {
   }
 
   render() {
-    const { errors } = this.props
+    const { errors, param } = this.props
+
 
     return (
       <div
         className='parameter-row-error'
-        tabIndex='-1'
+        tabIndex={errors.length ? -1 : 1}
```

Comment:
Why that change of `tabIndex?` 
In that case (if I'm not wrong) it will affect the natural flow of the document - forcing a focus on that item if there is no error.

Reply:
Ahh, okay.